scenarios/httpclient.benchmarks.yml

```diff
@@ -0,0 +1,44 @@
+﻿# Local run:
+# crank --config "C:\Users\knatalia\dev\git\Benchmarks\scenarios\httpclient.benchmarks.yml" --scenario httpclient-kestrel-get --profile local
+
+imports:
+  - https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml?raw=true
+
+variables:
+    serverPort: 5000
+    httpVersion: "1.1"
+    numberOfClients: 1
+    concurrencyPerClient: 12
+    http11MaxConnectionsPerServer: 1
+    http20EnableMultipleConnections: false
+    warmup: 15
+    duration: 30
+
+scenarios:
+  httpclient-kestrel-get:
+    application:
+      job: kestrel
+    load:
+      job: httpClient
+      variables:
+        scenario: get
+
+jobs:
+  kestrel:
+    source:
+      repository: https://github.com/CarnaViire/benchmarks.git #https://github.com/aspnet/benchmarks.git
+      #localFolder: C:\Users\knatalia\dev\git\Benchmarks
+      branchOrCommit: http3-benchmarks #main
+      project: src/BenchmarksApps/HttpClientBenchmarks/Servers/Kestrel/Kestrel.csproj
+    readyStateText: Application started # app should write this line to output! 
+    arguments: "--urls https://{{serverAddress}}:{{serverPort}}"
+
+  httpClient:
+    source:
+      repository: https://github.com/CarnaViire/benchmarks.git #https://github.com/aspnet/benchmarks.git
+      #localFolder: C:\Users\knatalia\dev\git\Benchmarks
+      branchOrCommit: http3-benchmarks #main
+      project: src/BenchmarksApps/HttpClientBenchmarks/Clients/HttpClient/HttpClient.csproj
+    isConsoleApp: true
```

Comment:
What does this setting do?

Reply:
It means that the app is "ready" from the start, so Crank would not wait for a signal from the app to consider it started -- as opposing to server which needs some time to start before a load can start attacking it. In case of server or any other app which need some time to start, you would need to set a different setting "readyStateText" and then output that line from the app to signal to Crank that app is ready.
https://github.com/dotnet/crank/blob/main/src/Microsoft.Crank.Controller/benchmarks.schema.json#L154